sys/cloudformation/stack.yaml

```diff
@@ -97,10 +100,18 @@ Parameters:
     ConstraintDescription: >
       VPC Id must begin with 'vpc-' or leave blank to have a new VPC created
 
+  EnvAPPDEBUG:
+    Type: String
+    Default: '0'
```

Comment:
You should add "AllowedValues" to enforce a boolean-like behaviour

Reply:
fixed